src/Stratis.Bitcoin.Features.Wallet/Models/WalletHistoryModel.cs

```diff
@@ -82,6 +82,7 @@ public class PaymentDetailModel
     public enum TransactionItemType
     {
         Received,
-        Send
+        Send,
+        Mined
```

Comment:
Should this be 'Staked' instead of 'Mined'? I know they are used interchangeably but I like to think of Stake as POS and Mine as POW to keep things straight or does this identify POW blocks as well?

Reply:
hmm you're right, although the QT says Mined.
Well, we're doing Staked! Thanks Mike.